lib/main.dart

```diff
@@ -26,7 +26,7 @@ Future<void> main() async {
     await settingsStorage.initialise();
     await PushNotificationService().initialise();
     await remoteConfigurations.initialise();
-    await TokenModel.installModels(['lightwallet','experimental'], [TokenModel.seedsEcosysUsecase]);
+    await TokenModel.installModels(['localscale','lightwallet','experimental'], [TokenModel.seedsEcosysUsecase]);
```

Comment:
So this adds localscale tokens - fine by me

Reply:
Yeah, but to be squeaky clean I also staged https://dao.hypha.earth/hypha/agreements/65804